bin/node/cli/Cargo.toml

```diff
@@ -100,6 +101,7 @@ tempfile = "3.1.0"
 build-script-utils = { version = "2.0.0", package = "substrate-build-script-utils", path = "../../../utils/build-script-utils" }
 structopt = { version = "0.3.8", optional = true }
 node-transaction-factory = { version = "0.8.0", optional = true, path = "../transaction-factory" }
+node-inspect = { version = "0.8.0", optional = true, path = "../inspect" }
```

Comment:
Why do we need it as build dependency?

Reply:
I was puzzled too, but it seems that in the `build.rs` we are building some bash-completion support and it requires duplicating all the deps as build deps.

Reply:
Okay :woman_shrugging: